x-pack/plugins/secops/public/components/markdown/__snapshots__/markdown_hint.test.tsx.snap

```diff
@@ -1,155 +1,7 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`MarkdownHint rendering it renders the expected hints 1`] = `
-.c1 {
-  font-weight: bold;
-  margin-right: 5px;
-}
-
-.c0 {
-  visibility: inline;
-}
-
-.c6 {
-  margin-left: 5px;
-}
-
-.c2 {
-  font-style: italic;
-  margin-right: 5px;
-}
-
-.c5 {
-  -webkit-text-decoration: line-through;
-  text-decoration: line-through;
-}
-
-.c3 {
-  font-family: monospace;
-  margin-right: 5px;
-}
-
-.c4 {
-  margin-right: 5px;
-}
-
-<pure(Component)
+<Component
```

Comment:
Does this snapshot being reduce to the following continue to provide value?

``` html
<Component
  show={true} />
```

Reply:
Hard to say? It's not annoying us anymore which is great!

If we have to take a step back again and see if these snapshots are helpful down the road it should at least be easier.